test/verify/check-metrics

```diff
@@ -748,6 +748,13 @@ class TestCurrentMetrics(MachineCase):
         with b.wait_timeout(180):
             b.wait(lambda: float(b.text("#load-avg .pf-l-flex div:first-child").split()[-1].rstrip(',')) < 10)
 
+        # Test link to user services
+        m.execute("su - admin -c 'export XDG_RUNTIME_DIR=/run/user/$(id -u admin); systemd-run --user --collect --slice cockpittest -p CPUQuota=60% --unit cpu-hog dd if=/dev/urandom of=/dev/null'")
+        b.wait_text("table[aria-label='Top 5 CPU services'] tbody tr:nth-of-type(2) td[data-label='Service']", "cpu-hog")
+        b.click("table[aria-label='Top 5 CPU services'] tbody tr:nth-of-type(2) td[data-label='Service'] a span")
```

Comment:
Let's wait until cpu-userhog makes it to the top, i.e. `tr:nth-of-type(1)` like above; the "cpu-piglet" uses less CPU, and thus we expected that in the second row.

Right now the test fails everywhere, e.g. on [fedora-35](https://logs.cockpit-project.org/logs/pull-17039-20220228-185055-ffceafe7-fedora-35-devel/log.html#293-1). The [screenshot](https://logs.cockpit-project.org/logs/pull-17039-20220228-185055-ffceafe7-fedora-35-devel/TestCurrentMetrics-testCPU-fedora-35-127.0.0.2-2301-FAIL.png) looks rather messy - there are two cpu-hog services, probably the old ones from above which already got stopped, but the UI didn't catch up yet. (That's the reason for giving them a new name). Also, user@1000 is still very busy, possibly the test case failed to stop some process in the session?